config/genomes_GRCh38_gencode.config

```diff
@@ -0,0 +1,13 @@
+params {
+  // illumina iGenomes reference file paths on UPPMAX
+  igenomes_base = '/sw/data/uppnex/igenomes/'
+ 
+  // illumina iGenomes reference file paths
+  genomes {
+    'GRCh38' {
+      gtf         = "${params.igenomes_base}/Homo_sapiens/NCBI/GRCh38/Annotation/Genes.gencode/genes.gtf"
+      bed12         = false
```

Comment:
haha super nitpick, can you align the equal signs? 🤓

Reply:
:D sure